CITING.md

```diff
@@ -119,15 +148,217 @@ articleno = {2},
 numpages = {26},
 keywords = {irregular matrices, GPUs, Sparse Matrix Vector Product (SpMV)}
 }
+
+@inproceedings{anztEvaluatingPerformanceNVIDIA2020,
+  title = {Evaluating the {{Performance}} of {{NVIDIA}}'s {{A100 Ampere GPU}} for {{Sparse}} and {{Batched Computations}}},
+  booktitle = {2020 {{IEEE}}/{{ACM Performance Modeling}}, {{Benchmarking}} and {{Simulation}} of {{High Performance Computer Systems}} ({{PMBS}})},
+  author = {Anzt, Hartwig and Tsai, Yuhsiang M. and Abdelfattah, Ahmad and Cojean, Terry and Dongarra, Jack},
+  year = {2020},
+  month = nov,
+  pages = {26--38},
+  doi = {10.1109/PMBS51919.2020.00009},
+  urldate = {2023-12-06},
+  keywords = {ginkgo}
+}
+```
+
+
+### Preconditioners
+
+```bibtex
+@inproceedings{gobelMixedPrecisionIncomplete2021,
```

Comment:
We could also consider Hartwig and Tobias' ParILUT paper in this section?
https://scholar.google.com/citations?view_op=view_citation&citation_for_view=ctNCF8YAAAAJ:RGFaLdJalmkC